### DISCUSSION TITLE
Sorry for an abolute idiotic question...

### DISCUSSION THREAD

**Discussion by astro-moose** [aaronwmorris/indi-allsky#823] [Q&A] [answered]
...from a linux newbie...

but how do i do an update to indi-allsky? Is it with apt-get update and so on or is there another approach?

regards, Gerhard

**Comment by astro-moose** [comment]
found ;-)

**Comment by aaronwmorris** [comment, marked as answer]
Just in case others see this topic, I have a wiki page on updating indi-allsky.

https://github.com/aaronwmorris/indi-allsky/wiki/Updating-indi-allsky
